BEIMA.Backend/MongoService/Device.cs

```diff
@@ -152,10 +197,10 @@ public void SetLocation(ObjectId buildingId, string notes, string latitude, stri
             latitude ??= string.Empty;
             longitude ??= string.Empty;
 
-            Location.Set("buildingId", buildingId);
-            Location.Set("notes", notes);
-            Location.Set("latitude", latitude);
-            Location.Set("longitude", longitude);
+            Location.BuildingId = buildingId;
```

Comment:
I'm assuming that the .ToBsonDocument still works properly with this when sending it into the mongo methods?

Reply:
Yeah it does